salt/kallithea-demo/configs/production.ini

```diff
@@ -0,0 +1,604 @@
+errormator.api_key = YOUR_API_KEY
+
+## TWEAK AMOUNT OF INFO SENT HERE
+
+## enables 404 error logging (default False)
+errormator.report_404 = false
+
+## time in seconds after request is considered being slow (default 1)
+errormator.slow_request_time = 1
+
+## record slow requests in application
+## (needs to be enabled for slow datastore recording and time tracking)
+errormator.slow_requests = true
+
+## enable hooking to application loggers
+# errormator.logging = true
+
+## minimum log level for log capture
+# errormator.logging.level = WARNING
+
+## send logs only from erroneous/slow requests
+## (saves API quota for intensive logging)
+errormator.logging_on_error = false
+
+## list of additonal keywords that should be grabbed from environ object
+## can be string with comma separated list of words in lowercase
+## (by default client will always send following info:
+## 'REMOTE_USER', 'REMOTE_ADDR', 'SERVER_NAME', 'CONTENT_TYPE' + all keys that
+## start with HTTP* this list be extended with additional keywords here
+errormator.environ_keys_whitelist =
+
+
+## list of keywords that should be blanked from request object
+## can be string with comma separated list of words in lowercase
+## (by default client will always blank keys that contain following words
+## 'password', 'passwd', 'pwd', 'auth_tkt', 'secret', 'csrf'
+## this list be extended with additional keywords set here
+errormator.request_keys_blacklist =
+
+
+## list of namespaces that should be ignores when gathering log entries
+## can be string with comma separated list of namespaces
+## (by default the client ignores own entries: errormator_client.client)
+errormator.log_namespace_blacklist =
+
+
+################
+### [sentry] ###
+################
+
+## sentry is a alternative open source error aggregator
+## you must install python packages `sentry` and `raven` to enable
+
+sentry.dsn = YOUR_DNS
+sentry.servers =
+sentry.name =
+sentry.key =
+sentry.public_key =
+sentry.secret_key =
+sentry.project =
+sentry.site =
+sentry.include_paths =
+sentry.exclude_paths =
+
+
+################################################################################
+## WARNING: *THE LINE BELOW MUST BE UNCOMMENTED ON A PRODUCTION ENVIRONMENT*  ##
+## Debug mode will enable the interactive debugging tool, allowing ANYONE to  ##
+## execute malicious code after an exception is raised.                       ##
+################################################################################
+set debug = false
+
+##################################
+###       LOGVIEW CONFIG       ###
+##################################
+
+logview.sqlalchemy = #faa
+logview.pylons.templating = #bfb
+logview.pylons.util = #eee
+
+#########################################################
+### DB CONFIGS - EACH DB WILL HAVE IT'S OWN CONFIG    ###
+#########################################################
+
+# SQLITE [default]
+sqlalchemy.db1.url = sqlite:///%(here)s/kallithea.db?timeout=60
```

Comment:
It depends. I don't know the purpose of Kallithea. If we deploy it for serious using, postgres obviously should be used. Who decides that?